pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -341,6 +360,24 @@ type PDScheduleConfig struct {
 	// Only used to display
 	// +optional
 	SchedulersPayload map[string]string `toml:"schedulers-payload" json:"schedulers-payload,omitempty"`
+
+	// StoreLimitMode can be auto or manual, when set to auto,
+	// PD tries to change the store limit values according to
+	// the load state of the cluster dynamically. User can
+	// overwrite the auto-tuned value by pd-ctl, when the value
+	// is overwritten, the value is fixed until it is deleted.
+	// Default: manual
+	// +optional
+	StoreLimitMode *string `toml:"store-limit-mode" json:"store-limit-mode,omitempty"`
```

Comment:
this config is in which version?

Reply:
4.0, the configuration would be marked after pd release 4.0.0 rc version.

Reply:
it is not in 3.1.0, why adding it in this 3.1.0 pr? we add it later.

Reply:
updated.